clients/java/exp-client/gradle/libs.versions.toml

```diff
@@ -0,0 +1,10 @@
+# This file was generated by the Gradle 'init' task.
+# https://docs.gradle.org/current/userguide/platforms.html#sub::toml-dependencies-format
+
+[versions]
+guava = "33.1.0-jre"
```

Comment:
Why do we import guava? Do we need this?

Reply:
Not sure, let me check.